src/status_im/ui/screens/subs.cljs

```diff
@@ -41,19 +41,26 @@
 (reg-sub :peers-count :peers-count)
 (reg-sub :mailserver-status :mailserver-status)
 
+(reg-sub :fetching?
+         (fn [db]
+           (get db :inbox/fetching?)))
+
 (reg-sub :offline?
          :<- [:network-status]
          :<- [:sync-state]
          (fn [[network-status sync-state]]
            (or (= network-status :offline)
                (= sync-state :offline))))
 
-(reg-sub :connection-problem?
-         :<- [:mailserver-status]
+(reg-sub :disconnected?
          :<- [:peers-count]
-         (fn [[mailserver-status peers-count]]
-           (or (= :disconnected mailserver-status)
-               (zero? peers-count))))
+         (fn [peers-count]
+           (zero? peers-count)))
+
+(reg-sub :mailserver-error?
+         :<- [:mailserver-status]
+         (fn [mailserver-status]
+           (#{:error :disconnected} mailserver-status)))
```

Comment:
Could use set as fn. Not sure it looks simpler, but at least shorter.

```
(reg-sub :mailserver-error?
         :<- [:mailserver-status]
         #{:error :disconnected})
```

Reply:
doesn't work